### PR TITLE
fix: Absolute imports

### PR DESCRIPTION
**Pull request by rafinskipg**
### What does this PR do?

add @lib @components @stores imports

include types as type definitions , and add standard unified way of importing them

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -13,7 +13,16 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "baseUrl": "./",
+    "paths": {
+      "@lib/*": ["lib/*"],
+      "@components/*": ["components/*"],
+      "@stores/*": ["stores/*"],
```
Comment (adamgoth) [inline tsconfig.json:21]: With `baseURL` specified to `./`, these shortcuts are actually the same as just using the actual path. `@lib/*` does the same thing as `lib/*`. So I'm not sure we need the shortcut with the `@`. Thoughts?

Comment (rafinskipg) [reply tsconfig.json:21]: have you tried it without the '@' ? i don't know if it works

**Review by KenG-pro — APPROVED** [review]

**Review by KenG-pro — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (adamgoth) [reply tsconfig.json:21]: Using `lib/* works the same as `@lib/*`. I just made sure. Adding `"baseUrl": "./"` in `tsconfig.json` makes it work.

Comment (rafinskipg) [reply tsconfig.json:21]: ok, so we replace all the @ stuff

**Review by tyler17 — APPROVED** [review]

**Pull request merged**